### PR TITLE
Alizeh shop

### LINKED ISSUE #3: shop page lofi

### PR DESCRIPTION
**Pull request by alizehzaidi**
Merge request

### CODE REVIEW

File: src/pages/shop/Shop.css [hunk 1]
```diff
@@ -0,0 +1,706 @@
...
+    align-self: stretch;
+}
+
+.night-reviews-container {
+    display: flex;
+    align-items: flex-start;
+    gap: 5px;
+}
+
+.night-stars-container {
```
Comment (stephanierongone) [inline src/pages/shop/Shop.css]: I would try to combine some of these classes into one, just to make your file a little more condensed!

File: src/pages/shop/Shop.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+                                    <p>"$55"</p>
+                                </div>
+                            </div>
+                            <div className="set-description">
+                                <div className="set-description-text">
+                                    <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."</p>
+                                </div>
+                                <div className="set-reviews-container">
+                                    <div className="set-stars-container">
+                                        <img className="star1-image" src={} alt="star 1" />
```
Comment (stephanierongone) [inline src/pages/shop/Shop.js]: making things like the stars container a reusable component could come in handy later down the road

**Review by stephanierongone — CHANGES_REQUESTED** [review]
good overall! just a couple small changes

File: src/pages/AboutUs.css
Comment (ananyar807) [inline src/pages/AboutUs.css:1]: did you mean to add the aboutUs css file to this branch?

**Pull request merged**